src/test/groovy/org/gradle/profiler/mutations/ApplyProjectDependencyChangeSetupMutatorTest.groovy

```diff
@@ -0,0 +1,85 @@
+package org.gradle.profiler.mutations
+
+import org.gradle.profiler.mutations.support.ProjectCombinations
+
+import static org.gradle.profiler.mutations.support.ProjectCombinationsSupport.createProjectCombinations
+
+class ApplyProjectDependencyChangeSetupMutatorTest extends AbstractMutatorTest {
+
+    ProjectCombinations combinations
+    String salt
+
+    def setup() {
+        combinations = createProjectCombinations(2, 1)
+        salt = combinations.getSalt()
+    }
+
+    def "throws exception is settings file does not exist"() {
+        def combinations = createProjectCombinations(2, 1)
+
+        when:
+        new ApplyProjectDependencyChangeSetupMutator(tmpDir.root, combinations)
+
+        then:
+        def e = thrown(IllegalStateException)
+        e.message == "No settings.gradle(.kts) file found in ${tmpDir.root}"
+    }
+
+    def "generates projects to gradle-profiler-generated-projects folder"() {
+        tmpDir.newFile("settings.gradle")
+        def mutator = new ApplyProjectDependencyChangeSetupMutator(tmpDir.root, combinations)
+
+        when:
+        mutator.beforeScenario(scenarioContext)
+
+        then:
+        File project1BuildScript = file(tmpDir.root, "gradle-profiler-generated-projects", "project-$salt-0", "build.gradle")
```

Comment:
I think in this case it'd be fine to simply use `/`, it works on Windows as well.

Reply:
I did not know that, will change, thank you